UI/Web/src/app/app.component.ts

```diff
@@ -139,9 +119,7 @@ export class AppComponent implements OnInit {
 
 
   setCurrentUser() {
-    const user = this.accountService.getUserFromLocalStorage();
-    this.accountService.setCurrentUser(user);
-
+    const user = this.accountService.currentUserSignal();
```

Comment:
I see it functioning in the webapp, but why are we not setting the current user in the account service anymore?

Reply:
For some reason, I thought this was the accountSerivice, I'm good now.